pywbem/_statistics.py

```diff
@@ -0,0 +1,352 @@
+#!/usr/bin/env python
+#
+# (C) Copyright 2003-2007 Hewlett-Packard Development Company, L.P.
+# (C) Copyright 2006-2007 Novell, Inc.
+#
+# This library is free software; you can redistribute it and/or
+# modify it under the terms of the GNU Lesser General Public
+# License as published by the Free Software Foundation; either
+# version 2.1 of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful, but
+# WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+# Lesser General Public License for more details.
+#
+# You should have received a copy of the GNU Lesser General Public
+# License along with this program; if not, write to the Free Software
+# Foundation, Inc., 675 Mass Ave, Cambridge, MA 02139, USA.
+
+"""
+The :class:`~pywbem.Statistics` class allows measuring the elapsed
+time of instrumented coded and keeping statistics on the measured times
```

Comment:
Grammar issues in this line.
Change line 22 to:

    time of WBEM operations and keeping statistics on the measured times.